models/connector.go

```diff
@@ -1,20 +1,24 @@
 package models
 
+// Connector Struct representing a connector from the config
 type Connector struct {
-	Type     string
-	ID       string
-	Tags     string
-	Active   bool
-	Server   string
-	Port     string
-	Login    string
-	Pass     string
-	File     string
-	From     string
-	Key      string
-	Image    string
-	Users    string
-	Commands []Command
-	Debug    bool
-	BotName  string
+	Type      string
+	ID        string
+	Tags      string
+	Active    bool
+	Server    string
+	Port      string
+	Login     string
+	Pass      string
+	File      string
+	From      string
+	Key       string
+	Secret    string
+	KeyValues map[string]string
+	Image     string
+	Users     string
+	Commands  []Command
+	Debug     bool
+	BotName   string
+	Filter    []string
```

Comment:
Can't the "Secret" and "Filter" be added as KeyValues? Filter could potentially be a comma separated list?